build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
@brianrob Good question.

As a quick aside: I was thinking of following up with a later PR that refactors this a bit, putting PGO into one of four mutually exclusive states: off, on, instrument, or auto (aka try to PGOize but silently fall back to non-PGO if counts are not available).

I think for master, "auto" makes the most sense--that way developers get the same optimizations that we have in release branches automatically, *unless* they opt out. This is in line with the current behavior (just with no way to force either on|off). Either way, the `skiprestoreoptdata` can (and I think should) remain independent.